Doc/c-api/exceptions.rst

```diff
@@ -753,43 +753,45 @@ the variables:
 +-----------------------------------------+---------------------------------+----------+
 | C Name                                  | Python Name                     | Notes    |
 +=========================================+=================================+==========+
-| :c:data:`PyExc_BaseException`           | :exc:`BaseException`            | \(1)     |
-+-----------------------------------------+---------------------------------+----------+
-| :c:data:`PyExc_Exception`               | :exc:`Exception`                | \(1)     |
-+-----------------------------------------+---------------------------------+----------+
```

Comment:
I feel BaseException and Exception are special enough to ignore alphabetical order.